src/components/Dialog.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    :class="
+      'fixed w-full px-3 md:px-0 h-full flex flex-col justify-center items-center bg-black bg-opacity-60 z-50 top-0 left-0 ' +
+      classname
```

Comment:
We need to distinguish static class from dynamic class

In Vue Project, we write like this:

```html
<div
  class="xxxx"
  :class="xxxx"
/>
```

Reply:
Greate idea